spec/models/events/review_spec.rb

```diff
@@ -53,27 +53,85 @@
-        review
-        review_turnaround = ReviewTurnaround.last
-        expect(review_turnaround[:value]).to eq(0)
-        expect(review_turnaround[:review_request_id]).to eq(review_request.id)
+    describe '#build_review_turnaround' do
+      context 'when a review is created' do
+        before { travel_to(Time.zone.today.beginning_of_day) }
+        let(:review_request) { create(:review_request) }
+        let(:review) { create :review, review_request: review_request }
+
+        it 'creates a review turnaround with the correct values' do
+          review
+          review_turnaround = ReviewTurnaround.last
+          expect(review_turnaround[:value]).to eq(0)
+          expect(review_turnaround[:review_request_id]).to eq(review_request.id)
+        end
+
+        it 'creates a review turnaround' do
+          expect { review }.to change { ReviewTurnaround.count }.from(0).to(1)
+        end
+
+        context 'when there is more than one review in a review request' do
+          let!(:second_review) { create :review, review_request: review_request }
+
+          it 'does not create review turnaround' do
+            expect { review }.to_not change { ReviewTurnaround.count }
+          end
+        end
+
+        context 'when the count of reviews is one' do
+          let(:review) { create :review }
+          it 'builds the review turnaround' do
+            expect(Builders::ReviewTurnaround). to receive(:call)
+            review
+          end
+        end
+      end
+    end
+
+    describe '#build_second_review_turnaround' do
+      let(:project) { create(:project, language: Language.first) }
+      let(:vita) { create(:user, login: 'santiagovidal') }
+      let(:santib) { create(:user, login: 'santib') }
+      let(:hernan) { create(:user, login: 'hdamico') }
+      let(:pr) { create(:pull_request, owner: vita, project: project) }
+
+      let(:rr) do
+        create(:review_request, owner: vita, reviewer: santib, project: project, pull_request: pr)
+      end
+
+      let!(:review) do
+        create(:review, owner: santib, project: project, pull_request: pr, review_request: rr)
+      end
+
+      let(:second_rr) do
+        create(:review_request, owner: vita, reviewer: hernan, project: project, pull_request: pr)
       end
 
-      it 'creates a review turnaround' do
-        expect { review }.to change { ReviewTurnaround.count }.from(0).to(1)
+      let(:second_review) do
+        create(:review,
+               owner: hernan, project: project, pull_request: pr, review_request: second_rr)
+      end
+
+      let(:third_review) do
+        create(:review,
+               owner: hernan, project: project, pull_request: pr, review_request: second_rr)
+      end
+
+      context 'when a different user make the second review' do
+        it 'builds the complete review turnaround' do
+          expect(Builders::CompletedReviewTurnaround)
+            .to receive(:call)
+          second_review
```

Comment:
not necessary to change it now, but I think it'd be more valuable a test like this
```ruby
expect { second_review }.to change(CompletedReviewTurnaround).by(1)
```

because it's testing behavior instead of implementation

Reply:
i will change it anyway, it has more sense.